README.md

```diff
@@ -53,24 +53,153 @@ import { CalendarHeatmap } from '@manufac/reactjs-calendar-heatmap';
   color={color}
   overview={overview}
   handler={print}
+  onTooltip={show}
+  onHideTooltip={hide}
 />
 ```
 
 ### Properties
 
-| Property | Usage                                                                | Default | Required |
-| :------- | :------------------------------------------------------------------- | :-----: | :------: |
-| data     | Time series data from max a year back                                |  none   |   yes    |
-| color    | Theme hex color                                                      | #ff4500 |    no    |
-| overview | Initial overview type (choices are: year, month, day)                |  year   |    no    |
-| handler  | Handler function is fired on click of a time entry in daily overview |  none   |    no    |
+<table>
+  <tr>
+    <th>Property</th>
+    <th>Interface</th>
+    <th>Usage</th>
+    <th>Default</th>
+    <th>Required</th>
+  </tr>
+  <tr> 
+    <td>
+      data
+    </td>
+    <td>
+      <pre>
+        interface CalendarHeatmapDatum {
+          date: string;
+          total: number;
+          details: { name: string; date: string; value: number }[];
+          summary?: { name: string; value: number }[];
+        }
+      </pre>
+    </td>
+    <td>
+      Time series data from max a year back  
+    </td>
+    <td>
+      none
+    </td>
+    <td>
+      yes
+    </td>
+  </tr>
+  <tr> 
+    <td>
+      color
+    </td>
+    <td>
+      <pre>
+        string
+      </pre>
```

Comment:
```suggestion
        `string`
```

will this work?

Reply:
![Uploading Screenshot from 2022-04-29 19-23-42.png…]()